cellprofiler/gui/pipelinecontroller.py

```diff
@@ -2531,7 +2531,7 @@ def duplicate_modules(self, modules):
             module_num = len(self.__pipeline.modules()) + 1
         for m in modules:
             module = self.__pipeline.instantiate_module(m.module_name)
-            module.module_num = module_num
+            module.set_module_num(module_num)
```

Comment:
It looks like this behaves differently in Python 3.
From module.py - `module_num = property(get_module_num, set_module_num)`
In Python 2 it looks like trying to assign module_num works fine, but in Python 3 it triggers the get_ command which raises an exception (as the module_number we're trying to set is still the default of -1?). Calling the set_ command directly seems to get it working and allows you to duplicate modules.